app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,11 @@
     <string name="top_track">Top Track</string>
     <string name="search">Search</string>
     <string name="pcm_music">PCM Music</string>
+    <string name="singer_name_demo">Join Harry</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.singer_name_demo` appears to be unused
 Unused resources make applications larger and slow down builds.